src/appengine/handlers/cron/project_setup.py

```diff
@@ -101,22 +101,22 @@ def job_name(self, project_name):
 # The order of templates is important here. Later templates override settings in
 # the earlier ones. An engine template may override vars set for a sanitizer.
 LIBFUZZER_ASAN_JOB = JobInfo('libfuzzer_asan_', 'libfuzzer', 'address',
-                             ['engine_asan', 'libfuzzer', 'prune'])
+                             ['libfuzzer', 'engine_asan', 'prune'])
 LIBFUZZER_MSAN_JOB = JobInfo('libfuzzer_msan_', 'libfuzzer', 'memory',
-                             ['engine_msan', 'libfuzzer'])
+                             ['libfuzzer', 'engine_msan'])
 LIBFUZZER_UBSAN_JOB = JobInfo('libfuzzer_ubsan_', 'libfuzzer', 'undefined',
-                              ['engine_ubsan', 'libfuzzer'])
+                              ['libfuzzer', 'engine_ubsan'])
 AFL_ASAN_JOB = JobInfo(
     'afl_asan_',
     'afl',
-    'address', ['engine_asan', 'afl'],
+    'address', ['afl', 'engine_asan'],
     minimize_job_override=LIBFUZZER_ASAN_JOB)
 NO_ENGINE_ASAN_JOB = JobInfo('asan_', 'none', 'address', [])
 
 LIBFUZZER_ASAN_I386_JOB = JobInfo(
```

Comment:
Can you please move this up, so that it is easier to not miss it, feels like this newline at line 115 makes it easy to forget.

Reply:
done.